test/plugin/base_test.rb

```diff
@@ -1652,12 +1656,10 @@ def setup_no_k8s_metadata_stubs
   def setup_cloudfunctions_metadata_stubs
     stub_metadata_request(
       'instance/attributes/',
-      "attribute1\nkube-env\ngcf_region\nlast_attribute")
-    stub_metadata_request('instance/attributes/kube-env',
-                          "ENABLE_NODE_LOGGING: \"true\"\n"\
-                          'INSTANCE_PREFIX: '\
-                          "gke-#{CLOUDFUNCTIONS_CLUSTER_NAME}-740fdafa\n"\
-                          'KUBE_BEARER_TOKEN: AoQiMuwkNP2BMT0S')
+      "attribute1\ncluster-name\nkube-env\ngcf_region\nlast_attribute")
+    stub_metadata_request('instance/attributes/cluster-name',
+                          CONTAINER_CLUSTER_NAME)
+    stub_metadata_request_permission_denied('instance/attributes/kube-env')
```

Comment:
Ditto here — let's have a `setup_cloudfunctions_metadata_stubs_no_kube_env` in addition to this and replicate the test.